BCStudentSoftwareDevTeam/lsf
# 344 fix labor history ordering

Issue:
The problem is in the sorting order, which is intended to be in reverse chronological order, meaning that the most future term should be displayed at the top. However, the sorting for all terms was incorrect and was instead being ordered by date modified which was improper for the form.

Changes:
- Created ordered values for the term code
- Recreated the ordering function inside of the term model" to "Created a method in the Term class to order objects by term
- Created tests 
- Recreated the ordering function inside of the term model
- Made sure that we were passing our formhistories to the term.sort_by_term method

Testing:
- switch to 344-fix-labor-history-ordering
- source setup.sh and ./database/reset_database.sh from-backup
- flask run
- Navigate to the student search  
- The page should display the labor status forms of the selected student from past terms in reverse order (newest to oldest)
- Also, run the test suite

Attached below is how a proper ordering should look like
![labor status form ssdt](https://github.com/BCStudentSoftwareDevTeam/lsf/assets/122579222/72217ed1-ddc7-43b9-bb19-cb53661a9d2d)
